python/py-inflect/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-inflect
+version             7.0.0
+license             MIT
+maintainers         nomaintainer
+
+description         Correctly generate plurals, singular nouns, ordinals, \
+                    indefinite articles\; convert numbers to words
+long_description    {*}${description}
+
+homepage            https://github.com/jaraco/inflect
+checksums           rmd160  8ce769569c102155c3fe04a07b1ebabccd894d16 \
+                    sha256  63da9325ad29da81ec23e055b41225795ab793b4ecb483be5dc1fa363fd4717e \
+                    size    70963
+
+python.versions     38 39 310 311 312
+python.default_version 312
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+                    port:py${python.version}-setuptools_scm
+    depends_lib-append \
+                    port:py${python.version}-pydantic \
+                    port:py${python.version}-typing_extensions
+
+    livecheck.type      none
```

Comment:
thisbis the default set by the "python" PortGroup and thus could/should be removed